docs/reference/contrib/table_block.rst

```diff
@@ -99,13 +100,15 @@ When defining a TableBlock, Wagtail provides the ability to pass an optional ``t
 Configuration Options
 ^^^^^^^^^^^^^^^^^^^^^
 
-Every key in the ``table_options`` dictionary maps to a `handsontable <https://handsontable.com/>`_ option. These settings can be changed to alter the behaviour of tables in Wagtail. The following options are available:
+The ``table_header_choice`` option determines if rows or columns are treated as headings. Valid values are ``row``, ``column``, ``both``, and ``neither``. The default is ``row``. This will cause the elements in the first row of your table to be rendered as ``th`` elements with ``scope = row``.
```

Comment:
The distinction isn’t that big in my mind but generally I only see these referred to as "headers".

```suggestion
The ``table_header_choice`` option determines if rows or columns are treated as headers. Valid values are ``row``, ``column``, ``both``, and ``neither``. The default is ``row``. This will cause the elements in the first row of your table to be rendered as ``th`` elements with ``scope = row``.
```